x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -238,15 +236,19 @@ the PKI realms that have been configured for delegation.
 To permit authentication delegation for a specific {es} PKI realm, start by
 configuring the realm for the usual case, as detailed in the
 <<pki-realm-for-direct-clients>>
-section. However, you must make sure that you have explicitly configured
+section. Note that you must explicitly configure a `truststore`
+(or, equivalently `certificate_authorities`) when configuring
+a PKI realm for delegation, as delegated PKI will not automatically
+make use of the network layer's trust settings. 
 a `truststore` (or, equivalently `certificate_authorities`) even though it is
```

Comment:
This section ended up a bit garbled after my suggestion got applied (Oh I wish :octocat: would support multiline suggestions).

Reply:
Nice catch again, thanks!